README.md

```diff
@@ -53,7 +91,8 @@ To use Horovod, make the following additions to your program:
 1. Run `hvd.init()`.
 
 2. Pin a server GPU to be used by this process using `config.gpu_options.visible_device_list`.
-    With the typical setup of one GPU per process, this can be set to *local rank*.
+    With the typical setup of one GPU per process, this can be set to *local rank*. In that case, first process on the
+    server will be allocated first GPU, second process will be allocated second GPU and so forth.
```

Comment:
allocated **the first** GPU, the second process will be allocated **the second** GPU **,** and so forth.